modules/search/instant-search/components/search-result-minimal.scss

```
@@ -3,32 +3,21 @@
 	margin: 1em 0;
 	position: relative;
 
+	h3 {
```

Just in case the theme has some aggressive h3 styling, let's increase the specificity of this rule:

```suggestion
	.jetpack-instant-search__search-result-title {
```

---

Let's do that, yeah!